src/RobotInterface/include/BipedalLocomotion/RobotInterface/ISensorBridge.h

```diff
@@ -0,0 +1,318 @@
+/**
+ * @file ISensorBridge.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_ROBOT_INTERFACE_ISENSOR_BRIDGE_H
+#define BIPEDAL_LOCOMOTION_ROBOT_INTERFACE_ISENSOR_BRIDGE_H
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <Eigen/Dense>
+
+#include <BipedalLocomotion/GenericContainer/Vector.h>
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+
+namespace BipedalLocomotion
+{
+namespace RobotInterface
+{
+
+/**
+ * Sensor bridge options
+ */
+struct SensorBridgeOptions
```

Comment:
How are you planning to use these options?

Reply:
It's just like a board of switches to turn on/off the processing during the initialization step. An instance of the SensorBridge will not consider attaching itself to sensor streams that are not enabled.

Reply:
It makes sense but I did not see it used in the interface. What would be your plan?

Reply:
The intention was that the Derived class will hold private instances of `SensorBridgeOptions` and `SensorLists` for internal organization. I shall add a line of documentation on how it can be used.

Reply:
Ok. In any case, my suggestion would be to declare what you use and leave freedom to the user for the rest. If you declare some struct in the base interface, but the base interface does not use them (neither as an attribute nor as an input parameter to a method), then there is a big chance they are never used, or used in a wrong way 😁

Reply:
I wanted to have them within the RobotInterface namespace. Would it be better if I moved it to a new header keeping it separate from the interface?

Reply:
Do they use it in the same way? My main point is that either it is clear from the code how to use these structs, or it may be better to leave the freedom to the user.

A suggestion I may give is to have a function that automatically fills this struct given a pointer to a parameter handler, for example.

Reply:
I can add a protected method in the interface as per your suggestion.

Reply:
Sounds good!

Reply:
I added optional helper methods in https://github.com/dic-iit/bipedal-locomotion-framework/pull/87/commits/4a4b818f9bb3ad844993a7a6596c16844e7af942 to avoid the structs from remaining dangling.
I added a line of documentation which will allow the user to also avoid using these structs if they decide to implement their own better ways.